glide-core/src/request_type.rs

```diff
@@ -214,6 +214,7 @@ pub enum RequestType {
     XAutoClaim = 203,
     XInfoGroups = 204,
     XInfoConsumers = 205,
+    XInfoStream = 207,
     Scan = 206,
```

Comment:
this ordering is unacceptable!

Reply:
Lol 206 was stolen from me

Reply:
you could've kept the numbers while swapping the ordering 😆  - welp, too late.